recipe/recipe.yaml

```diff
@@ -0,0 +1,87 @@
+schema_version: 1
+
+context:
+  name: p4p
+  version: 4.2.0
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  - url: https://github.com/epics-base/${{ name }}/archive/refs/tags/${{ version }}.tar.gz
+    sha256: 2b89d7605ae35dc7edad2a36cf6b1d395170698b6d7fdfc3811a8d3ed5183bab
+    patches:
+      - patch/fix_sys_libs.patch
+      - patch/update_setup_config.patch
+      - patch/handle_no_default_asyncio_loop.patch
+  - path: src
+
+build:
+  number: 8
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --ignore-installed
+  python:
+    entry_points:
+      - pvagw = p4p.gw:main
+
+requirements:
+  build:
+    - make
+    - perl
+    - ${{ compiler('cxx') }}
+    - ${{ compiler('c') }}
+    - ${{ stdlib('c') }}
+    - if: build_platform != target_platform
+      then: python
+    - if: build_platform != target_platform
+      then: cross-python_${{ target_platform }}
+    - if: build_platform != target_platform
+      then: cython
+    - if: build_platform != target_platform
+      then: numpy
```

Comment:
I would put these in a single list.

Reply:
Aha, but we do it otherwise on epics-base: https://github.com/conda-forge/epics-base-feedstock/blob/af63da70aa7bcdd63186d7954e7518b0ea77bda4/recipe/recipe.yaml#L55-L60

Anyhow, I can accept this, but I think it's a lot more fragile and error-prone to remember that those are all the same condition. Auto-converted conversion tools preferences can largely be superseded.